simphony_ui/openfoam_boundary_conditions.py

```diff
@@ -0,0 +1,149 @@
+from traits.api import HasStrictTraits, Float, Enum, Instance, Array
+from traitsui.api import View, Item, Tabbed, UItem, VGroup
+
+
+class BoundaryConditionModel(HasStrictTraits):
+
+    #: The type of the boundary condition
+    type = Enum('none', 'empty', 'zeroGradient', 'fixedGradient', 'fixedValue')
+
+    traits_view = View(
+        VGroup(
+            'type',
+            Item(
+                name='fixed_gradient',
+                visible_when='type == "fixedGradient"',
+            ),
+            Item(
+                name='fixed_value',
+                visible_when='type == "fixedValue"',
+            ),
+            show_border=True,
+        )
+    )
+
+
+class VelocityBoundaryConditionModel(BoundaryConditionModel):
+
+    #: A fixed value of velocity gradient for the boundary
+    fixed_gradient = Array(Float, (1, 3))
```

Comment:
Well, for an array of float it's less problematic. If the user gives a negative float or an integer, it will still be considered as a float.

But I still should fix it because I don't instantiate the Array properly. The array constructor takes a dtype, not a trait.
So if I fix it with:
```python
fixed_gradient = Array('f', (1, 3))
```
Is it ok for you?